### DISCUSSION TITLE
mish activation function.

### DISCUSSION THREAD

**Discussion by Nightbringers** [google/flax#3391] [Ideas]
please add mish activation function.

**Comment by chiamp** [comment]
Flax imports activation functions from Jax. Adding the mish activation function was [proposed](https://github.com/google/jax/pull/1676#issuecomment-555785233), but ultimately Jax wanted to keep `jax.nn` very minimal. You could try raising the issue again to see if they're open to adding it, or you can implement it yourself as a [custom](https://github.com/google/jax/pull/1676/files#diff-fd212165604ca06ce1815fd67f2c48d8cb425f27922437d2ae523676f8be6caeR66-R72) activation function.
